cnf-tests/.konflux/rpms.in.yaml

```diff
@@ -0,0 +1,32 @@
+contentOrigin:
+  # Define at least one source of packages, but you can have as many as you want.
+  repofiles:
+     - ./redhat.repo
+     - ./ubi.repo
+
+packages:
+  # list of rpm names to resolve
+  # - openshift-clients # this needs the ocp RPM repo enabled
+  - bc
+  - linuxptp
+  - iperf3
+  - realtime-tests
+
+#reinstallPackages: []
+  # list of rpms already provided in the base image, but which should be
+  # reinstalled
+
+arches:
+  # The list of architectures can be set in the config file. Any `--arch` option set
+  # on the command line will override this list.
+#  - aarch64
+  - x86_64
+  # - s390x
+  # - ppc64le
+
+context:
+    # Alternative to setting command line options. Usually you will only want
+    # to include one of these options, with the exception of `flatpak` that
+    # can be combined with `image` and `containerfile`
+    containerfile: Containerfile
```

Comment:
There is no `Containerfile` in this folder. Do we need to change this to `Dockerfile`?
Also, I'm not getting where this `rpms.in.yaml` file is referenced from. Can we get rid of this?

Reply:
The containerfile was local indeed and we needed rpms.in.yaml to generate the rpms.lock file, however konflux instructions is to upload these to the source repo. I'll double-verify anyway. Thanks for your comment.

Reply:
testing in #2209

Reply:
we could use any docker/container file that has the final image as ubi, I replaced this to use the cnf-tests Dockerfile. The process of updating the rpms.lockfile.yaml with more deps if needed will be ironed and automatic updates to lockfiles as per konflux team as we advance so it would be easier to do updates to the deps.